packages/libzip/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://libzip.org/
+TERMUX_PKG_DESCRIPTION="library for reading, creating, and modifying zip archives."
+TERMUX_PKG_VERSION=1.5.1
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=47eaa45faa448c72bd6906e5a096846c469a185f293cafd8456abb165841b3f2
+TERMUX_PKG_SRCURL=https://libzip.org/download/libzip-${TERMUX_PKG_VERSION}.tar.gz
+
+termux_step_post_extract_package() {
+    termux_setup_cmake
```

Comment:
Why?

Reply:
Not sure I think I did a copy/paste from another package and did not search why, should be fixed in https://github.com/termux/termux-packages/pull/3433/commits/1a5b7fbaa7517efec2696bc9a16db0daeb4c2b13